backend/src/packages/users/user.repository.ts

```diff
@@ -106,8 +107,10 @@ class UserRepository implements Repository {
     return Promise.resolve(null);
   }
 
-  public delete(): ReturnType<Repository['delete']> {
-    return Promise.resolve(true);
+  public async delete(id: number): Promise<boolean> {
+    const rowsDeleted = await this.userModel.query().deleteById(id);
```

Comment:
Let's add a soft deletion

Reply:
Hi @v0ldemar01 , I've updated all the other requested changes. However, I'm not entirely sure on what needs to be done for this particular task. Could you please provide more detailed instructions on what I need to do?

Reply:
Soft deletion means that instead of deleting entity the deleted_at would be added

Reply:
The same one is on miniproject